robot/web/web.go

```diff
@@ -220,9 +235,12 @@ type webService struct {
 	mu           sync.Mutex
 	r            robot.Robot
 	rpcServer    rpc.Server
+	liteServer   *googlegrpc.Server
```

Comment:
I initially thought about trying to modify rpc.Server to support the unix socket stuff directly, so I could simply start two of those, and not maintain a separate concept, but... On the module side, I believe directly using google's Server keeps things easier to implement for third parties.

Reply:
I think I agree. Just need to establish the security considerations around it.

Reply:
This has been reworked so that a new moduleServer object is used that fulfills rpc.Server, so no extra registration is required.